src/utils/environment.h

```diff
@@ -71,6 +71,9 @@ class Env {
     // get Primitive Cache M
     int getPrimitiveCacheM() { return primitiveCacheM; }
 
+    // get KV Cache Location
+    int getKVCacheLocation() { return primitiveCacheM; }
```

Comment:
should be kvCacheLocation? and initKVCacheLocation is never called?

Reply:
fixed